### PR TITLE
aws_gcp_vpn: Initial commit

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

File: aws_gcp_vpn/main.tf [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  bgp_asn    = var.gcp_private_asn
+  ip_address = each.value.ip_address
+  type       = "ipsec.1"
+
+  tags = {
+    Name = "gcp-${var.gcp_project_id}-to-aws-cg${each.value.id}"
+  }
+}
+
+# FIXME This only needs to be created once per AWS account
```
Comment (jasonthomas) [inline aws_gcp_vpn/main.tf:78]: If it's once per account then maybe outside of this module and add it as a documentation requirement? We just need to aws_vpn_gateway.id as an input.

Comment (jbuck) [reply aws_gcp_vpn/main.tf:78]: mmm yeah

Comment (jasonthomas) [reply aws_gcp_vpn/main.tf:78]: Or maybe just a resource that can be flagged on or off.

**Review by jasonthomas — DISMISSED** [review]
r+wc

**Review by jasonthomas — APPROVED** [review]

**Pull request merged**